.changeset/lovely-shoes-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+[README] Update code sample to use uppercase `GET`.
```

Comment:
I don’t think we generally use the `[]` syntax for stuff, so I’d personally use a sentence form, but it’s not super important. LGTM!

```suggestion
Update code sample in the README to use uppercase `GET`.
```